scripts/githooks/check-file-format.sh

```diff
@@ -75,8 +68,32 @@ function is-arg-true() {
 
 # ==============================================================================
 
+check=${check:-working-tree-changes}
+
+case $check in
+    "all")
```

Comment:
One thing I noticed when I browsed this file in my IDE is that there are 4-space indentations in this section. The interesting thing is that VSCode gives me a visual indication of it but doesn't validate it, nor does the editorconfig, despite the setting "indent_size = 2." This is a bit of surprising behaviour.

![image](https://github.com/nhs-england-tools/repository-template/assets/499338/952d388f-f140-4429-add4-e12e601f45cd)

Reply:
Ugh, that's annoying.